docs/standards/smart-contracts/introduction.md

```diff
@@ -5,95 +5,6 @@ sidebar_position: 1
 
 # Smart Contract Implementation
 
-:::success Documentation
-
-The smart contracts are public and open source. They can be found [on GitHub](https://github.com/lukso-network/lsp-smart-contracts) and in the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) NPM package.
-
+:::info
+This page has been moved to a new section: [**Contracts -> Introductions**](../../contracts/introduction.md)
```

Comment:
You have set a redirect, you can simply delete this page.

Delete all old pages and use redirects from the config file as much as possible.

don't do both